docs/src/pages/index.tsx

```diff
@@ -22,50 +13,73 @@ import MembraneLogo from '@site/static/img/membrane-logo.svg';
 import WebGpuLogoDark from '@site/static/img/webgpu-dark.svg';
 import WebGpuLogoLight from '@site/static/img/webgpu-light.svg';
 import { useColorMode } from '@docusaurus/theme-common';
+import TypewriterComponent from 'typewriter-effect';
 
 function HomepageHeader() {
   return (
-    <header className={clsx('hero hero--primary', styles.heroBanner)}>
-      <div className="container">
-        <div className="row">
-          <div className="col col--2"></div>
-          <div className="col col--8">
-            <div className="container">
-              <Heading as="h2" className="hero__title">
-                The best solution for programmatic live video compositing.
-              </Heading>
-              <p>
-                Process live video streams in real-time. Apply effects, combine videos, or add
-                transitions to produce any output you want.
-              </p>
-              <div className="row">
-                <Link
-                  className="button button--secondary button--lg col col-6 margin--sm"
-                  to="/docs/intro">
-                  Docs
-                </Link>
-                <Link
-                  className="button button--secondary button--outline button--lg col col-6 margin--sm"
-                  to="https://github.com/membraneframework/live_compositor">
-                  View on GitHub
-                </Link>
-              </div>
+    <div className={clsx('container', styles.firstSection)}>
+      <div className="row">
+        <div className="col col--6">
+          <div className="container">
+            <Heading as="h1" style={{ fontSize: 55 }}>
+              <span className="text--primary">Mix video and audio</span>
+              <TypewriterComponent
```

Comment:
This cause periodic layout change on some devices. Can we avoid it?

https://github.com/membraneframework/live_compositor/assets/104033489/e2f4f578-35ce-4027-83e3-a729d9243843